src/pykeen/models/uncertainty.py

```diff
@@ -4,22 +4,21 @@
 Analyze uncertainty.
 
 Currently, all implemented approaches are based on Monte-Carlo dropout [gal2016]_.
-
-Monte-Carlo dropout relies on the model having dropout layers. While dropout usually is 
-turned off for inference / evaluation mode, MC dropout leaves dropout enabled. Thereby, 
+Monte-Carlo dropout relies on the model having dropout layers. While dropout usually is
+turned off for inference / evaluation mode, MC dropout leaves dropout enabled. Thereby,
 if we run the same prediction method $k$ times, we get $k$ different predictions.
 The variance of these predictions can be used as an approximation of uncertainty, where
 larger variance indicates higher uncertainty.
 
-Example:
+The following code-block sketches an example use case:
```

Comment:
This seems to be related to https://github.com/PyCQA/pydocstyle/issues/412

Reply:
just let me know when you're done and I will take care of all remaining code style stuff